shielddash/api/models.py

```diff
@@ -1,8 +1,34 @@
 from django.db import models
 
 
-class Retention(models.Model):
+class Study(models.Model):
+    name = models.CharField(max_length=255, primary_key=True)
```

Comment:
Would we use this as the PK or would we use an `id`? I was expecting an `id` and we'd make `name` unique.